include/hffix.hpp

```diff
@@ -74,6 +74,12 @@ inline void throw_range_error() {
 template <std::size_t N>
 std::ptrdiff_t len(char const (&)[N]) { return std::ptrdiff_t(N - 1); }
 
+std::size_t pow(std::size_t v, std::size_t exp)
```

Comment:
in C++11 I would have made this ``constexpr``, but I'm pretty confident the compiler will constant fold this function away anyway